### DISCUSSION TITLE
Reflect the latest CSS spec for pointer cursor usage

### DISCUSSION THREAD

**Discussion by ryo-manba** [adobe/react-spectrum#6305] [General]
Hi Team,
React Aria generally avoids using the pointer cursor on non-link elements as described in the [FAQ](https://github.com/adobe/react-spectrum/wiki/Frequently-Asked-Questions-(FAQs)#why-dont-buttons-use-a-pointer-cursor), we avoid using a pointer cursor for non-link elements. However, the latest [CSS specification](https://drafts.csswg.org/css-ui/#valdef-cursor-pointer) states that pointer cursors can be used for other interactive elements. As noted in [this comment](https://github.com/w3c/aria-practices/issues/2951#issuecomment-2041578354), APG follows this latest specification.

Are there any plans for React Aria to follow this decision as well?

**Comment by reidbarber** [comment]
That section of the FAQ is specific to React Spectrum. Our examples in React Aria tend to follow that too, mainly for consistency. I don't think we prevent anyone from adding the pointer cursor anywhere in React Aria though. That is up to the implementer, and I think using the pointer cursor would be completely valid choice.

**Comment by devongovett** [comment]
"Can" doesn't mean "should". 🙂

**Comment by ryo-manba** [comment]
Got it, that makes sense now. Thanks for the clarification!
